src/funcs/getColumnDefinition.js

```diff
@@ -1,7 +1,7 @@
 const getColumnType = require('./getColumnType');
 const getColumnSettings = require('./getColumnSettings');
 
-module.exports = function getColumnDefinition(col) {
+module.exports = function getColumnDefinition(col, primaryKeys) {
```

Comment:
This is just a drive-by at this point. I thought I'd need to pass some info about the available `enum` types in here, and then noticed that `primaryKeys` [was being passed in](https://github.com/polarislabs/pg-to-dbml/blob/450ea07b86526654c69e95b239e3075310d9ef45/src/funcs/transformTableStructureToDBML.js#L10) as the 2nd parameter already.